torchx/c_src/torchx.cpp

```diff
@@ -368,6 +368,16 @@ NIF(flip)
   TENSOR(torch::flip(*input, dims));
 }
 
+NIF(unfold)
+{
+  TENSOR_PARAM(0, input);
+  PARAM(1, int64_t, dim);
+  PARAM(2, int64_t, size);
+  PARAM(3, int64_t, step);
+
+  TENSOR(at::native::unfold(*input, dim, size, step));
```

Comment:
I still need to check if this function is supported on GPU. It wasn't documented and I found it by chance looking from the Python perspective of libtorch.

Reply:
All tests pass with `LIBTORCH_TARGET=cu111`, so it works!